sensors/2.0/iiohal_mediation_v2.0/libiio_client/iiod-client.c

```diff
@@ -40,11 +42,13 @@ static ssize_t iiod_client_read_integer(struct iiod_client *client,
     ssize_t ret;
     int value;
 
-    do {
-        ret = client->ops->read_line(client->pdata,
-                desc, buf, sizeof(buf));
-        if (ret < 0)
-            return ret;
+	do {
+		ret = client->ops->read_line(client->pdata,
```

Comment:
Check for indentation, removed lines and add lines are not following same spaces.